mapper/mapper.go

```diff
@@ -264,6 +265,18 @@ func CreateGetCoverage(req *http.Request, basePage coreModel.Page, lang, filterI
 		results = append(results, result)
 	}
 
+	totalPages := pagination.GetTotalPages(areas.TotalCount, areas.Limit)
+	var paginatedResults coreModel.Pagination
```

Comment:
according to language guidelines package names should typically not be mixedCaps/underscores etc https://go.dev/blog/package-names

Reply:
Are you talking about `coreModel` or `pagination` here - if `coreModel` it is well out of scope for this ticket